contact.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Data Stewardship Wizard</title>
+    <link href="./_css/appendix.css" rel="stylesheet">
+  </head>
+  <body>
+		<div class="public">
+      <nav class="navbar navbar-expand-sm bg-primary fixed-top">
+        <div class="container">
+          <div class="navbar-header">
+            <a class="navbar-brand" href="/">
+              <img src="/img/dsw-logo.svg" alt="DSW logo">
+              Data Stewardship Wizard
+            </a>
+          </div>
+        </div>
+      </nav>
+	  <div class="container">
+		<div class="Public_BookReference">
+			<h2>Contact</h2>
+				<table class="contact-table">
+				<tbody>
+					<tr>
+						<th>Address</th>
+						<td>Uppsala University, SciLifeLab Data Centre<br>
+							Dept. of Immunology, Genetics and Pathology<br>
+							Husargatan 3, 751 08 Uppsala<br>
+							Sweden
+						</td>
+					</tr>
+					<tr>
+						<th>Email</th>
+						<td><a href="mailto:datacentre@scilifelab.se">datacentre@scilifelab.se</a>
+						</td>
+					</tr>
+					<tr>
+						<th class="contact-table-th">Bug reporting</th>
```

Comment:
Maybe add a comment about issues in the SciLifeLab knowledge models to be reported to our github repository?

Reply:
So, have info about two issue reports, one general dsw, and one SLL KM?

Reply:
Exactly; I don't think they would be happy to get reports about misstakes in our KMs.

Reply:
Which link do I use to SLL KM?

Reply:
https://github.com/ScilifelabDataCentre/dsw-issues

Just like the issues reported directly from inside DSW.